ui/src/app/dashboard/grouped-summary/grouped-summary.component.ts

```diff
@@ -13,28 +13,30 @@ const DEFAULT_NUM_ROW = 5;
 enum CardStatus {
   EXPANDED = 'expanded',
   COLLAPSED = 'collapsed',
-  NONE = 'none'
+  NONE = ''
 }
 
 @Component({
   selector: 'jm-grouped-summary',
   templateUrl: './grouped-summary.component.html',
   styleUrls: ['./grouped-summary.component.css']
 })
-export class GroupedSummaryComponent implements OnInit{
+export class GroupedSummaryComponent implements OnInit {
   @Input() aggregation: Aggregation;
   @Input() statusArray: Array<JobStatus>;
 
-  displayedAggregationEntries = [];
-  originalAggregationEntries = [];
+  displayedAggregationEntries = new Array<Map<JobStatus, Number>>();
```

Comment:
Really these are maps of strings to strings (or maybe string|number) aren't they? You put a "label" key into this map (not a JobStatus), and the values are strings or numbers